test/conversion_reaction/conditions.tsv

```diff
@@ -0,0 +1,2 @@
+conditionId
```

Comment:
the conversion reaction example is already under doc/example. A second one should not be needed.